### PR TITLE
docs: add release notes for web 13sep

### PR DESCRIPTION
**Pull request by raviteja83**

### CODE REVIEW

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 1]
```diff
@@ -12,6 +12,23 @@ description: Release Notes for 100ms JavaScript SDK
...
+
+#### Added:
+- `join` is now async
+- A `HMSNotificationTypes.PEER_LIST` will be sent after join even if there are no peers already in the room
+- An error will be returned if `validateVideoPluginSupport` is called before video track is available
+- Added facingMode property for local videoTrack.(Note: This will be useful on mobile browsers)
+
+#### Fixed:
+- Cleanup on leave
+- Improved logging for beam
```
Comment (raviteja83) [inline docs/javascript/v2/release-notes/release-notes.mdx:27]: should we make it as browser recording instead of beam?

Comment (triptu) [reply docs/javascript/v2/release-notes/release-notes.mdx:27]: yes

File: docs/javascript/v2/features/join.mdx [hunk 2]
```diff
@@ -35,7 +37,7 @@ const config = {
     rememberDeviceSelection: true,  // remember manual device change
 };
 
-hmsActions.join(config);
+await hmsActions.join(config);
```
Comment (triptu) [inline docs/javascript/v2/features/join.mdx:40]: Does this require changes in quickstart too?

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 3]
```diff
@@ -12,6 +12,23 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/react-sdk                   | [![npm version](https://badge.fury.io/js/%40100mslive%2Freact-sdk.svg)](https://badge.fury.io/js/%40100mslive%2Freact-sdk)             |
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
+## 2022-09-13
+
+Released: `@100mslive/hms-video-store@0.6.0`, `@100mslive/react-sdk@0.4.0`
+
+#### Added:
+- `join` is now async
+- A `HMSNotificationTypes.PEER_LIST` will be sent after join even if there are no peers already in the room
```
Comment (triptu) [inline docs/javascript/v2/release-notes/release-notes.mdx:21]: Should go in breaking

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 4]
```diff
@@ -12,6 +12,23 @@ description: Release Notes for 100ms JavaScript SDK
...
+- An error will be returned if `validateVideoPluginSupport` is called before video track is available
+- `facingMode` property for local videoTrack.(Note: This will be useful on mobile browsers)
```
Comment (triptu) [inline docs/javascript/v2/release-notes/release-notes.mdx:23]: Let's link the word facingMode  to corresponding mdn docs?

Comment (raviteja83) [reply docs/javascript/v2/release-notes/release-notes.mdx:23]: yes, will add

**Review by triptu — APPROVED** [review]

**Pull request merged**